server/graphql/v2/input/CommentCreate.js

```diff
@@ -0,0 +1,16 @@
+import { GraphQLString, GraphQLInt, GraphQLInputObjectType } from 'graphql';
+
+/**
+ * Input type to use as the type for the comment input in createComment mutation.
+ */
+const CommentCreate = new GraphQLInputObjectType({
+  name: 'CommentCreate',
+  fields: () => ({
+    markdown: { type: GraphQLString },
+    html: { type: GraphQLString },
+    ExpenseId: { type: GraphQLInt },
+    UpdateId: { type: GraphQLInt },
```

Comment:
I kind of missed that but one of the design goals of the GraphQL API v2 is to stop exposing directly internal integers. We should think about an alternative.

Reply:
🤔 this looks like the one that got away 🙂.

Let me create a PR to change it.

Also I would like to do a little bit of refactoring to use some of the functions I introduced in the PR.